Library/Include/Babylon/Console.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include <napi/napi.h>
+
+#include <sstream>
+
+namespace Babylon
+{
+    template<typename LogHandlerT>
```

Comment:
This pattern of passing in an instance to the log handler seems weird to me. The caller must guarantee that the instance live longer than this class. I think what we should do is pass in 3 handlers template args for Log, Warn, and Error and default Warn and Error to Log.